File: docs/examples/unit_tests/author_solution/python2/Buyer.py
class Buyer:
	def __init__(self, cash=0):
		self.cash = cash

	def getCash(self):
		return self.cash

	def setCash(self,cash=0):
		self.cash = cash

	def buy(self, seller):
		if self.cash > 0:
			if seller.sell() == 1:
				self.cash -=1

File: tests/deployment_tests/data/docker_locale/Buyer.java
import java.io.*;
class Buyer
{
  private int cash;

  public Buyer(int cash)
  {
    this.cash=cash;
  }
  // Special characters included to verify docker locale settings.
 //Ѓ ջ ݚ ৬ ౙ ඥ
  public int getCash()
  {
    return cash;
  }

  public void setCash(int cash)
  {
    this.cash=cash;
  }

  public void buy(Seller x)
  {
    if(cash>0)
    {
      if(x.sell()==1)
      {
        cash--;
      }
    }
  }
}
